src/perf.rs

```diff
@@ -10,6 +10,50 @@ use crate::table::Table;
 use crate::types::*;
 use crate::BccError;
 
+// TODO:
+//    - The enums below should probably generated in bcc-sys
+//    - Find a way to map generated to
+
+#[allow(dead_code)]
+pub enum PerfType {
```

Comment:
This enum doesn't look like it covers the other types, we definitely want to support the other types of perf events here too. We have current use cases for the hw cache type and the raw type is likely to be useful in the near term